Assets/Mirror/Editor/EditorHelper.cs

```diff
@@ -0,0 +1,31 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Mirror.EditorScripts
+{
+    public static class EditorHelper
```

Comment:
Mirror.EditorScripts.EditorHelper is very verbose.
Why not Mirror.Editor and then Mirror.Editor.Utils/Helpers?

Reply:
Editor shouldn't be used as a namespace because it is a UnityEditor class.

I can just drop the extra part and go with `namespace Mirror` since that seems to be what the rest of the editor classes here

Reply:
yes please